web/styles/zulip.css

```diff
@@ -393,6 +393,12 @@ p.n-margin {
 }
 
 #gear-menu .dropdown-menu {
+    /*
+    If there is sufficient space, the `.dropdown-menu` will always have a height
+    based on its content. If the screen is small, it will have a height of 85vh.
+    */
+    max-height: 85vh;
+
```

Comment:
This value will fail in various scenarios and is hard to maintain. We should use `message_viewport.height() ` - header height - navbar_alerts_wrapper height.

Reply:
How can we get those values in the CSS? Via custom properties?